### PR TITLE
Add rating

### PR DESCRIPTION
**Pull request by ChrysanthemumT**
just some minimal rating system

### CODE REVIEW

**Comment by ChrysanthemumT** [comment]
idk if this is still relevant after our meeting

File: src/main/java/seedu/address/model/person/Name.java [hunk 1]
```diff
@@ -64,4 +64,6 @@ public int hashCode() {
         return fullName.hashCode();
     }
 
+    public static class Rating {
+    }
```
Comment (jerremyng) [inline src/main/java/seedu/address/model/person/Name.java:68]: Remove redundant code.
```suggestion

```

File: src/main/java/seedu/address/model/person/Person.java
Comment (jerremyng) [inline src/main/java/seedu/address/model/person/Person.java:1]: We also need to add this to the Person constructor and integrate it to the app. But good start to rating system.

**Review by jerremyng — COMMENTED** [review]
LGTM, but lets push this aside till further discussion.

**Comment by ChrysanthemumT** [comment]
fixed
please review and merge

**Review by jerremyng — COMMENTED** [review]
LGTM

**Pull request merged**